packages/dev/src/examples/networks-and-flows/graph/graph-selections/index.tsx

```diff
@@ -4,7 +4,7 @@ import { Graph } from '@unovis/ts'
 import { generateNodeLinkData, NodeDatum, LinkDatum } from '@src/utils/data'
 
 export const title = 'Basic Graph'
-export const subTitle = 'Generated Data'
+export const subTitle = 'Select Node on Click'
```

Comment:
@rokotyan Would you mind explaining how we should be labeling title, subTitle, and category? I want to be consistent but it's not clear to me the function of each one.

Reply:
@reb-dev I would like them to give us a better understanding of what particular example is about. I don't have a clear guidance for that yet, so it's something we'll need to crystallize over time. If you have ideas, you're more than welcome to share them!


In the particular case:
<img width="204" alt="Screen Shot 2023-01-20 at 12 04 27 PM" src="https://user-images.githubusercontent.com/755708/213795988-52073e42-bb6d-4724-b7c4-2d18568ef93a.png">
`category` tells us that it's a Graph example (I think this field will almost always be related to the component name);
`title` tells us that i'll be a rather basic example;
but `subTitle` give us an idea that there's something extra, i.e. node selection logic.